server/routes/approvingVariations/handlers/varyApproveList.ts

```diff
@@ -22,11 +22,11 @@ export default class VaryApproveListRoutes {
         const licence = _.head(c.licences)
 
         const releaseDate = c.nomisRecord.releaseDate
-          ? format(parse(c.nomisRecord.releaseDate, 'yyyy-MM-dd', new Date()), 'dd MMM yyyy')
+          ? format(parseIsoDate(c.nomisRecord.releaseDate), 'dd MMM yyyy')
```

Comment:
we shouldn't be doing parsing of dates that we've retrieved from backend data sources at all 😱
Once we've removed some caseload logic bloat from the frontend we can look at pulling this down into the data layer